src/pe_source/data/helpers/redact_pii.py

```diff
@@ -0,0 +1,560 @@
+
+    type = "WA_drivers_license"
+
+
+class WA_DLDetector(scrubadub.detectors.RegexDetector):
+    """Create detector class to identify WA drivers licenses."""
+
+    name = "WA_drivers_license"
+    regex = re.compile("|".join(WA), re.IGNORECASE)
+    filth_cls = WA_DLFilth
+
+
+class WV_DLFilth(scrubadub.filth.Filth):
+    """Create filth class for WV drivers licenses."""
+
+    type = "WV_drivers_license"
+
+
+class WV_DLDetector(scrubadub.detectors.RegexDetector):
+    """Create detector class to identify WV drivers licenses."""
+
+    name = "WV_drivers_license"
+    regex = re.compile("|".join(WV), re.IGNORECASE)
+    filth_cls = WV_DLFilth
+
+
+class WI_DLFilth(scrubadub.filth.Filth):
+    """Create filth class for WI drivers licenses."""
+
+    type = "WI_drivers_license"
+
+
+class WI_DLDetector(scrubadub.detectors.RegexDetector):
+    """Create detector class to identify WI drivers licenses."""
+
+    name = "WI_drivers_license"
+    regex = re.compile("|".join(WI), re.IGNORECASE)
+    filth_cls = WI_DLFilth
+
+
+class WY_DLFilth(scrubadub.filth.Filth):
+    """Create filth class for WY drivers licenses."""
+
+    type = "WY_drivers_license"
+
+
+class WY_DLDetector(scrubadub.detectors.RegexDetector):
+    """Create detector class to identify WY drivers licenses."""
+
+    name = "WY_drivers_license"
+    regex = re.compile("|".join(WY), re.IGNORECASE)
+    filth_cls = WY_DLFilth
+
+
+# Build a detector to find Social security numbers with no spaces
+class SSNFilth(scrubadub.filth.Filth):
+    """Create filth class for Social Security numbers."""
+
+    type = "no_space_social_security_number"
+
+
+class SSNDetector(scrubadub.detectors.RegexDetector):
+    """Create detector class to identify social security numbers."""
+
+    name = "no_space_ssn"
+    regex = re.compile(
+        r"(?:(?<=\s)|(?<=^))(social security number|Social Security No|Social Security #|social security number|social|Social|ssn)\W*(?!219099999|078051120)(?!666|000|9\d{2})\d{3}(?!00)\d{2}(?!0{4})\d{4}(?=$|\s)",
+        re.IGNORECASE,
+    )
+    filth_cls = SSNFilth
+
+
+# Build a detector that finds passport numbers based off of previous context
+class PassportFilth(scrubadub.filth.Filth):
+    """Create filth class for passport numbers."""
+
+    type = "passport"
+
+
+class PassportDetector(scrubadub.detectors.RegexDetector):
+    """Create detector class to identify passport numbers."""
+
+    name = "passport"
+    regex = re.compile(
+        r"(Passport Number|Passport No|Passport #|Passport#|PassportID|Passportno|passportnumber)\W*\d{9}",
+        re.IGNORECASE,
+    )
+    filth_cls = PassportFilth
+
+
+# Build a detector that identifies Alien Id numbers
+class AlienIdFilth(scrubadub.filth.Filth):
+    """Create filth class for alien identification numbers."""
+
+    type = "alien id"
+
+
+class AlienIdDetector(scrubadub.detectors.RegexDetector):
+    """Create detector class to identify alien identification numbers."""
```

Comment:
Looks like the official term for these are [Alien Registration Numbers](https://www.uscis.gov/tools/glossary).
```suggestion
# Build a detector that identifies Alien Registration Numbers
class AlienIdFilth(scrubadub.filth.Filth):
    """Create filth class for Alien Registration Numbers."""

    type = "alien id"


class AlienIdDetector(scrubadub.detectors.RegexDetector):
    """Create detector class to identify Alien Registration Numbers."""
```

I'll leave it up to you to rename the class names  and modify the `"alien id"` strings appropriately.

Reply:
@dav3r issue is closed in the following commit: [6e10627](https://github.com/cisagov/pe-reports/pull/190/commits/6e106273ced7e5d9d93bd4ddbb3762b2de5f5391)